packages/types/src/provider/api.ts

```diff
@@ -234,7 +234,7 @@ export interface ProviderRegistered {
 
 export type ApiJsonError = {
 	message: string;
-	code: number;
+	code: string | number;
```

Comment:
is the string a code in this situation? e.g. "500". If so, can we parseInt() to keep a single type

Reply:
sorted